libraries/standard/mqtt/src/mqtt.c

```diff
@@ -163,7 +165,36 @@ MQTTStatus_t MQTT_Publish( MQTTContext_t * const pContext,
 
 MQTTStatus_t MQTT_Ping( MQTTContext_t * const pContext )
 {
-    return MQTTSuccess;
+    int32_t bytesSent = 0;
+    MQTTStatus_t status = MQTTSuccess;
+
+    if( pContext == NULL )
+    {
+        IotLogError( "pContext is NULL." );
+        status = MQTTBadParameter;
+    }
```

Comment:
There are missing else blocks. Did we run MISRA on this?

Reply:
MISRA 15.7 reports missing else block only for if-else-if blocks. Yes MISRA checks were run on this.